app/controllers/application_controller.rb

```diff
@@ -1234,15 +1234,15 @@ def redirect_to_next_object(method, model, id)
                                 }, query))
   end
 
-  private ##########
-
   def find_query_and_next_object(object, method, id)
     # prev/next in RssLog query
     query_and_next_object_rss_log_increment(object, method) ||
       # other cases (normal case or no next object)
       query_and_next_object_normal(object, method, id)
   end
 
+  private ##########
```

Comment:
It depends on how you count. I added calls to test/controllers/name_controller.rb in order to get coverage and it wouldn't let me do that if it was declared private.